build/transpile.js

```diff
@@ -606,7 +608,8 @@ class Transpiler {
             [ / \+\= (?!\d)/g, ' .= ' ],
             [ /([^\s\(]+(?:\s*\(.+\))?)\.toUpperCase\s*\(\)/g, 'strtoupper($1)' ],
             [ /([^\s\(]+(?:\s*\(.+\))?)\.toLowerCase\s*\(\)/g, 'strtolower($1)' ],
-            // [ /([^\s\(]+(?:\s*\(.+\))?)\.replaceAll\s*\(([^)]+)\)/g, 'str_replace($2, $1)' ], // still not a part of the standard in Node.js 13
+            [ /([^\s\(]+(?:\s*\(.+\))?)\.trim\s*\(\)/g, 'trim($1)' ],
+            [ /([^\s\(]+(?:\s*\(.+\))?)\.replaceAll\s*\(([^)]+)\)/g, 'str_replace($2, $1)' ],
```

Comment:
@ttodua hum, I think we will get this error in PHP
`Fatal error: Uncaught ArgumentCountError: str_replace() expects at least 3 arguments, 2 given in`

Reply:
even though that line contains `'str_replace($2, $1)'`, however there the regex result `$2` includes two words separated by comma, so in the end there should be 3 args.  did you run some example that threw the error?

Reply:
i.e. https://regex101.com/r/WdqKRE/1

Reply:
> even though that regex line shows `'str_replace($2, $1)'`, the regex result `$2` includes two words separated by comma, so after transpilation there should be 3 args. did you run some example that threw the error?

@ttodua right, I did not notice the group was capturing both arguments, thanks for checking it